fe/fe-core/src/main/java/com/starrocks/connector/CatalogConnectorMetadata.java

```diff
@@ -149,6 +151,28 @@ public Table getTable(ConnectContext context, String dbName, String tblName) {
         return metadata.getTable(context, dbName, tblName);
     }
 
+    @Override
+    public TvrTableSnapshot getCurrentTvrSnapshot(String dbName, Table table) {
```

Comment:
what does Tvr mean? time-varying relation?  how about using TVR instead of Tvr, Ivm instead of IVM

Reply:
yes. it means for `time-varying relation` Since this is just a choose for IVM's implementation.

> `how about using TVR instead of Tvr`?
Since `Tvr` is already used for current codes, can we keep to use it `Tvr` then?